build/brew-template.rb

```diff
@@ -18,9 +19,14 @@ def install
     doc.install resource( "apidocs" )
   end
 
+  test do
```

Comment:
Curious what do these lines do?  Will this actually start a server when the user installs it?